source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
+======================
+
+.. _pr-standard:
+
+Purpose
+-------
+
+The primary purpose of a review is to make sure that the overall health of MongoDB's
+corpus is improving over time. All of the tools and processes of review are
```

Comment:
Comment: I've never heard "corpus" used to describe our content outside of this document. Would it make sense to choose a different word?

I've heard "corpus" only used in the context of computational linguistic for training models. Maybe "content", "collected writings", "docuntation"? Not sure of a better word.